scripts/install_staging.sh

```diff
@@ -0,0 +1,82 @@
+# Update your system packages
+sudo apt update 
+
+# Install additional required libraries and tools
+sudo apt install --assume-yes make build-essential git clang curl libssl-dev llvm libudev-dev protobuf-compiler
+
+# Install rust and cargo
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+
+# Update your shell's source to include Cargo's path
+source "$HOME/.cargo/env"
+
+# Clone subtensor and enter the directory
+cd ../
+git clone https://github.com/opentensor/subtensor.git
+cd subtensor
+
+# Update to the nightly version of rust
+./scripts/init.sh
+
+# Initialize your local subtensor chain in development mode. This command will set up and run a local subtensor network.
+# Run this in the background (or send to tmux/forked shell?)
+
+# Check if inside a tmux session
+# Start a new tmux session and create a new pane, but do not switch to it
+tmux new-session -d -s localnet -n 'script' 'bash ./scripts/localnet.sh'
+# Notify the user
+echo "localnet.sh is running in a detached tmux session named 'localnet'"
+echo "You can attach to this session with: tmux attach-session -t localnet"
+
+
+# Navigate to your project directory
+cd ../bittensor-subnet-template
+
+# Install the bittensor-subnet-template python package
+python -m pip install -e .
+
+# Create a coldkey for the owner role
+btcli wallet new_coldkey --wallet.name owner2
```

Comment:
we can split the file into multiple parts:
- `install.sh`: takes care of dependencies and installation
- `create.sh`: creates wallets, registers subnet, everything up until `btcli wallet overview`
- `demo.sh`: runs demo (tmux)